frontend/src/app/workflows/model/GenerateResult.ts

```diff
@@ -0,0 +1,10 @@
+export interface GenerateResult {
```

Comment:
I think the numberOf*** variables are kind of redundant, why don't you get the size of the map instead?

Reply:
Agreed